README.md

```diff
@@ -0,0 +1,127 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  
+  <br/>
+
+  <h3><b>Welcome...</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [PORTFOLIO] <a name="about-project"></a>
+
+My PORTFOLIO project in Microverse.
+
+**[PORTFLIO]** is a project experience that Microverse offers to help nuture your skills in designing on small screens.  
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+ 1. HTML5
+ 2. CSS3
+
+
+### Key Features <a name="key-features"></a>
+
+- **[prints out a web page of a portfolio]**
+
```

Comment:
- Hey, let's make this more descriptive by adding the key feature you've worked on. e.g.: The portfolio mobile version section. you can choose to add more. please, update this to make your readme more professional.